packages/amazon-sumerian-hosts-babylon/src/Babylon.js/HostObject.js

```diff
@@ -5,6 +5,8 @@ import anim from './animpack';
 import aws from './awspack';
 import PointOfInterestFeature from './PointOfInterestFeature';
 
+const AWS = require('aws-sdk');
```

Comment:
Why are we using the `require()` syntax here instead of the `import` syntax? If `import` syntax is an option (which I assume it is) we should use that to be consistent.

Reply:
I thought I had reached the conclusion that the `require` syntax was necessary for modules to be able to access each other's global variables, but I can't find proof of this assumption (and, well, it wouldn't make them global...) 

I'll switch this to `import` to match the other statements, it makes no difference to the way this file uses this module.